### DISCUSSION TITLE
Sponsored Setup Q

### DISCUSSION THREAD

**Discussion by qtw1998** [squidfunk/mkdocs-material#3058] [Q&A]
Thx for your excellent jobs! Love it and  I paid the monthly payment, then how could I use the insiders version 
waiting for your help bro

**Comment by qtw1998** [comment]
> Thx for your excellent jobs! Love it and I paid the monthly payment, then how could I use the insiders version
> waiting for your help bro

`Collecting git+https://****@github.com/squidfunk/mkdocs-material-insiders.git
  Cloning https://****@github.com/squidfunk/mkdocs-material-insiders.git to /tmp/pip-req-build-_n2nuk5y
  Running command git clone -q 'https://****@github.com/squidfunk/mkdocs-material-insiders.git' /tmp/pip-req-build-_n2nuk5y
  remote: Repository not found.
  fatal: repository 'https://github.com/squidfunk/mkdocs-material-insiders.git/' not found
WARNING: Discarding git+https://****@github.com/squidfunk/mkdocs-material-insiders.git. Command errored out with exit status 128: git clone -q 'https://****@github.com/squidfunk/mkdocs-material-insiders.git' /tmp/pip-req-build-_n2nuk5y Check the logs for full command output.
ERROR: Command errored out with exit status 128: git clone -q 'https://****@github.com/squidfunk/mkdocs-material-insiders.git' /tmp/pip-req-build-_n2nuk5y Check the logs for full command output.`

**Reply by squidfunk** [reply]
Thanks for sponsoring! The [Insiders getting started guide](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/) contains a step-by-step guide which should answer all of your questions.
